docs/overrides/main.html

```diff
@@ -26,3 +26,10 @@
 <link rel="icon" class="js-site-favicon" type="image/svg+xml" href="{{ config.theme.favicon | url}}">
 <meta name="generator" content="mkdocs-{{ mkdocs_version }}, mkdocs-material-9.1.21">
 {% endblock %}
+
+{% block outdated %}
+  You're not viewing the latest version.
+  <a href="{{ '../' ~ base_url }}">
+    Click here to go to latest.
```

Comment:
Nit: @gautierronan do you prefer "latest" or "stable"? I have a slight preference for "latest" as I find it more descriptive (it's what JAX is using), but for example NumPy and PyTorch use "stable". 🤷‍♂️

Reply:
Yes I changed to "stable" because that was used by Numpy and Pytorch, and I also prefer it a bit on my end. I prefer "latest" a bit less, but wouldn't mind it either.

Reply:
Also, while changing to "stable", I found that the banner now appears when using the url `.../stable/...` but not when using `.../0.2/...` so not sure why. That was not the case before (it only appeared on `.../dev/...`, as expected).

Reply:
Ok, let's go for stable :) We can maybe edit this banner text then.

> Also, while changing to "stable", I found that the banner now appears when using the url .../stable/... but not when using .../0.2/... so not sure why. That was not the case before (it only appeared on .../dev/..., as expected).

Weird, did you do the `mike set-default stable`?

Reply:
> Weird, did you do the mike set-default stable?
Yes I did. Are you seeing the same problem locally as well?